base/applications/rapps/include/appinfo.h

```diff
@@ -8,17 +8,17 @@
 enum LicenseType
 {
     LICENSE_NONE,
-    LICENSE_OPENSOURCE,
-    LICENSE_FREEWARE,
-    LICENSE_TRIAL,
+    LICENSE_OPENSOURCE = 1,
+    LICENSE_FREEWARE = 2,
+    LICENSE_TRIAL = 3,
     LICENSE_MIN = LICENSE_NONE,
     LICENSE_MAX = LICENSE_TRIAL
 };
 
 inline BOOL
-IsLicenseType(INT x)
+IsKnownLicenseType(INT x)
 {
-    return (x >= LICENSE_MIN && x <= LICENSE_MAX);
+    return (x > 0 && x <= LICENSE_MAX);
```

Comment:
```suggestion
    return (x > LICENSE_MIN && x <= LICENSE_MAX);
```
or maybe `x > LICENSE_NONE` ?

Reply:
If it was up to me, LICENSE_NONE would have been -1 but it is too late for that now, these numbers are set forever and hardcoding 0 is fine.